.github/workflows/build.yml

```diff
@@ -96,3 +96,28 @@ jobs:
 
       - name: Upload Code Coverage
         uses: codecov/codecov-action@v3
+
+  other:
+    name: Other
```

Comment:
Could you please change it to something like `cargo-deny`, so that it's clearer on the CI UI, what is actually run.

Reply:
Done.